tsconfig.json

```diff
@@ -20,6 +20,7 @@
 
     /* Advanced Options */
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
-    "sourceMap": true
+    "sourceMap": true,
+    "incremental": true
```

Comment:
used this opportunity to enable incremental typechecking for our repository, but we should bring it to every modular repository by default

Reply:
Check https://github.com/vercel/next.js/issues/8708#issuecomment-570384682 for why `incremental` doesn't make sense in the context of a project that uses Babel to transpile TypeScript.

Reply:
It's still useful while running typechecking locally (and can confirm `yarn tsc` is noticeably faster)

Reply:
Perhaps if it's solely for typechecking, then `--incremental` can be used instead? https://github.com/vercel/next.js/issues/14997#issuecomment-726692314 Mostly depends on what else uses `tsconfig.json` and whether the presence of `incremental` can get in the way of that.